### DISCUSSION TITLE
libFLAC - change after update

### DISCUSSION THREAD

**Discussion by stfmdc** [staxrip/staxrip#1395] [Q&A]
Hi,

In v2.39.3 are libFLAC v1.4.3 but in latest StaxRip v.2.40.0 libFLAC version is too old; v1.3.1

I mean on libFLAC in eac3to --- Apps\Audio\eac3to

Why? :D

**Comment by Dendraspis** [comment]
This version should be the same since `v2.38`.
In order to avoid issues I kept the one that is shipped with `eac3to`.

**Reply by stfmdc** [reply]
I don't know how, but now it's v1.4.3 again... :)
I just restart my PC for some Windows update and now it's previous version of libFLAC, weird...
